src/irr/asset/CGraphicsPipelineLoaderMTL.cpp

```diff
@@ -1,5 +1,59 @@
 #include "irr/asset/CGraphicsPipelineLoaderMTL.h"
```

Comment:
why is the loader in the front-end includes?

Reply:
i dont understand. u mean in `include/`? it's not, irrlicht target just have `src/` in include search dirs, it's nothing new

Reply:
I see.